.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "third_party/SchemaStore/schemastore"]
 	path = third_party/SchemaStore/schemastore
 	url = https://github.com/SchemaStore/schemastore.git
+[submodule "third_party/WebAssembly/threads"]
```

Comment:
By the way, feel free to do a first PR with just the submodule. You'll also have to update `scripts/sync.sh` and make sure the entry is sorted, but both of those are caught by tests.